Remove duplicated global symbol, controls[] is already defined in koules.c

Fixes this link error:
/usr/bin/ld: server.o:(.bss+0x10000): multiple definition of `controls'; koules.o:(.bss+0x7d160): first defined here

diff --git a/server.c b/server.c
index 5463244..e8e80f4 100644
--- a/server.c
+++ b/server.c
@@ -80,7 +80,6 @@ static long     sizes[MAXPACKET];
 static long     isizes[MAXPACKET];
 char            acceled[5];
 #endif
-struct control  controls[MAXROCKETS];
 static unsigned char Clientmap[5];
 static int      Socket;
 /*static int      tbreak; */
